docs/source/author/topics.rst

```diff
@@ -613,11 +613,14 @@ Curriculum Areas Configuration File
 
       - ``name:`` The name of the curriculum area (this is what will be displayed to the
         user).
+      - ``colour:`` The CSS colour class to use for colouring the curriculum
+        area badge on the website.
+        This colour is also applied to all children of curriculum area.
```

Comment:
didn't we define all our own colours, how does an author see the list of available colours when picking the colour to use in the config file?